xml/ay_bigfile.xml

```diff
@@ -4490,6 +4490,42 @@ config:type="boolean"&gt;true&lt;/diag&gt;</screen>
    </para>
 
    <sect2 xml:id="Software.Selections.sles10">
+    <title>Product Selection</title>
+    <para>
+     Starting with SLE-15 all products are distributed using one media. You've to
```

Comment:
"You have to choose" (the contraction is only used when there is another verb)

Reply:
i accepted, but to learn something ... I thought that "have to" and "choose" are verbs, so could you give an example? Thx